packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -102,6 +108,24 @@ class EmbeddedContentConsumer extends React.Component {
       this.props.onMount(this.xfcFrame);
     }
 
+    // Cover other scenario where xfc frame style doesn't apply
+    // such as when `srcdoc` attribute is used which doesn't work
+    // within xfc's JSONRPC communication.
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('focus', () => {
+      const styleObj = parse(focusStyleStr);
+      Object.entries(styleObj).forEach(([key, value]) => {
+        this.xfcFrame.iframe.style[key] = value;
+      });
+    }, true);
+
+    // Listen for blur event and callback function to apply the style
+    this.xfcFrame?.iframe?.contentWindow?.addEventListener('blur', () => {
+      const styleObj = parse(blurStyleStr);
+      Object.entries(styleObj).forEach(([key, value]) => {
+        this.xfcFrame.iframe.style[key] = value;
+      });
+    }, true);
```

Comment:
Instead of this logic, you should be able to use the ":focus" psuedo class.

Reply:
Can you give an example on how to setup the style and apply to the iframe's style?

Reply:
Like this?

```
iframe:focus {
  outline: 2px dashed #000;
}
```

Reply:
I would think that your iFrame would have an associated class name for styling.  However, it would be a similar concept to what you have described here, yes.

Reply:
The :focus pseudo-class isn't working for iframe. I'll check to see if we can update the classname on the fly for switching the style.